### PR TITLE
Codenvy 1266 packaging Docs into war

### PR DESCRIPTION
**Pull request by vkuznyetsov**
### What does this PR do?

This PR needed for https://github.com/codenvy/codenvy/issues/1266

### CODE REVIEW

**Comment by TylerJewell** [comment]
If you are moving the location of _site, then can you make an update to docs.sh file for building and running the docs site in dev mode?

**Comment by vzhukovskii** [comment]
Set a normal title to the pull request

**Comment by vkuznyetsov** [comment]
PR updated, @TylerJewell docs.sh works as usually for building/running

File: docs/_config.yml [hunk 1]
```diff
@@ -39,4 +39,7 @@ exclude:
   - /dev/
   - README.md
   - .gitignore
-  - docs.sh
\ No newline at end of file
+  - docs.sh
+  - assembly/*
+  - target/*
+  - pom.xml
```
Comment (skabashnyuk) [inline docs/_config.yml]: new line

File: docs/assembly/build_war.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+markdown: kramdown
+highlighter: rouge
+destination: target/_site
+#gauges_id: 503c5af6613f5d0f19000027
+#google_analytics_id: UA-50755011-1
+#google_site_verification: onQcXpAvtHBrUI5LlroHNE_FP0b2qvFyPq7VZw36iEY
```
Comment (skabashnyuk) [inline docs/assembly/build_war.yml]: remove if not used

File: docs/assembly/build_war.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+repository: https://github.com/codenvy/codenvy
+#help_url: https://github.com/jekyll/jekyll-help
```
Comment (skabashnyuk) [inline docs/assembly/build_war.yml]: remove if not used

File: docs/assembly/build_war.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+logo: /img/logo-2x.png
+
+gems:
+  - jekyll-feed
+  - jekyll-redirect-from
+  - jekyll-sitemap
+#  - jekyll-mentions
+#  - jekyll-avatar
+#  - jekyll-seo-tag
+#  - jemoji
```
Comment (skabashnyuk) [inline docs/assembly/build_war.yml]: remove if not used

File: docs/assembly/build_war.yml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+exclude:
+  - README.md
+  - .gitignore
+  - docs.sh
+  - assembly/*
+  - target/*
+  - pom.xml
```
Comment (skabashnyuk) [inline docs/assembly/build_war.yml]: new line

Comment (vkuznyetsov) [inline docs/assembly/build_war.yml]: PR updated

**Pull request merged**